docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,432 @@
+.. currentmodule:: discord
+
+
+Errors and Error Handling
+==========================
+
+Handling Errors
+---------------
+
+When an error ocurrs in am ext.commands command, a client event, or a bot listener, it will by default show an error in the console 
+to denote that the error ocurred. If you wish to handle errors a different way, an error handler is needed.
+
+In :ref:`Commands <discord_ext_commands>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Error handlers are called for every exception that occurs within a command's body, :func:`before invoke <discord.ext.commands.before_invoke>` and 
+:func:`after invoke <discord.ext.commands.after_invoke>` hooks, :ref:`command converters <ext_commands_converters>`, :ref:`command checks <ext_commands_checks>`, etc.
+
+These errors inherit from :exc:`~.ext.commands.CommandError`. If an error that occurs during the execution of a command does not inherit from :exc:`~.ext.commands.CommandError`, 
+it will automatically get wrapped in a :exc:`~.ext.commands.CommandInvokeError`, to retrieve/unwrap this error we need to access the error's ``.original`` attribute.
```

Comment:
```suggestion
it will automatically get wrapped in a :exc:`~.ext.commands.CommandInvokeError`; to retrieve/unwrap this error we need to access the error's ``.original`` attribute.
```